### ISSUE TITLE
Hiro wallet lost subdomain name display in v3.33.0 for firefox, chrome

### ISSUE THREAD

**Issue by paradigma-cl** [blockstack/blockstack-app#3075]
The latest version does not display the subdomain name associated with the Stacks address.  Reappears Account1 
When managing multiple accounts based on subdomains from one phrase, the reference is lost. Please, if we can recover this functionality

**Comment by markmhendrickson** [comment]
Are these subdomains registered from Stacks 1.0 or 2.0?

**Comment by paradigma-cl** [comment]
Stacks 2.0

**Comment by paradigma-cl** [comment]
Example:  The Hiro Wallet before yesterday used to display the user as phillip.xck.app .  Today, Account1

**Comment by badonyx** [comment]
Probably broken due to #3077. See also https://github.com/hirosystems/stacks-blockchain-api/issues/1540.

**Comment by markmhendrickson** [comment]
@paradigma-cl are these names still missing for you? I believe there was an API-side issue affecting this that may be resolved now.

**Comment by paradigma-cl** [comment]
> @paradigma-cl are these names still missing for you? I believe there was an API-side issue affecting this that may be resolved now.

Yes, they are back now!!!! Thank you very much.  This functionality is becoming very important for the growing number of our customers that are using did with subdomains to use the application.  They come from the business sector, and they are very demanding sometimes.

**Comment by badonyx** [comment]
Re: #3077.

BNSX is [supposed to provide](https://docs.bns.xyz/integrate-bnsx/bnsx-api) a "backwards-compatible interface from the Stacks Blockchain API's BNS endpoints" but it doesn't appear to have implemented any way to handle names with status `registered_subdomain`. I haven't really seen these documented well anywhere. I guess they were implemented in https://github.com/stacks-network/subdomain-registrar/pull/42. 

https://api.bns.xyz/v1/names/phillip.xck.app
```
{"error":"cannot find name phillip.xck","isBnsx":false}
```

https://stacks-node-api.mainnet.stacks.co/v1/names/phillip.xck.app
```
{
  "address": "SP1ZHGD5N96421HBVB41G444TEZR70KDGTTBGNXNN",
  "blockchain": "stacks",
  "last_txid": "0xf5d82720a592e89a61fc8dbe060db232d57930e44d5cc0ae52f22c92d24146bd",
  "resolver": "",
  "status": "registered_subdomain",
  "zonefile": "$ORIGIN phillip.xck.app.\n$TTL 3600\n_http._tcp\tIN\tURI\t10\t1\t\"https://gaia.blockstack.org/hub/1v16mBUaV6o4aaHj9xgXo9x5CBz68cHh8/profile.json\" \n\n",
  "zonefile_hash": "7aeb76c0fa1678b750747376ff4f604f"
}
```

**Comment by markmhendrickson** [comment]
@paradigma-cl glad to hear it!

@0xbabo this `registered_subdomain` behavior seems related to what's going on with the Stacks Blockchain API re: https://github.com/hirosystems/stacks-blockchain-api/issues/1540#issuecomment-1420152161 

I wonder if @rafaelcr has any insight here

**Comment by rafaelcr** [comment]
I believe the original subdomain problem might be more related to this Stacks node issue than to an API issue: https://github.com/stacks-network/stacks-blockchain/issues/3270

If the Stacks node used by the Hiro API does *not* have the specified zonefile attachment (where subdomains are declared) then that name will disappear from our responses. We've recently switched environments in our deployment which is probably why @paradigma-cl can see that name again i.e. the node on the new env does have that zonefile available.

**Comment by paradigma-cl** [comment]
I am glad that this feature of the subdomains is considered for the updates after the 2.1 launches.   The subdomain usage will also grow tremendously in the coming months.
> I believe the original subdomain problem might be more related to this Stacks node issue than to an API issue: [stacks-network/stacks-blockchain#3270](https://github.com/stacks-network/stacks-blockchain/issues/3270)
>
